### ISSUE TITLE
Feature request: default `popupId` to `useId`

### ISSUE THREAD

**Issue by eugene-mohc** [jcoreio/material-ui-popup-state#128]
I would like to suggest a feature to either remove the `popupId` property or keep it optional but always default it to the `useId` hook.

The purpose of the `useId` hook is exactly for cases such as these, and it would cut down some boilerplate of always having to manually specify `popupId: useId()` every time.

**Comment by tusbar** [comment]
I’m not sure there’s even a reason why you’d ever want to pass a non-generated id.

**Comment by jedwards1211** [comment]
Makes sense, I'll see if I can implement this soon, would also be open to a PR.

I've generally used explicit IDs so that I can select the popup component in Selenium tests, though attaching data- attributes for test selectors would work too
